storage/disk/example_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2021 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package disk_test
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/open-policy-agent/opa/storage"
+	"github.com/open-policy-agent/opa/storage/disk"
+	"github.com/open-policy-agent/opa/util"
+)
+
+func check(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+func Example_store() {
+
+	ctx := context.Background()
+
+	// Create a temporary directory for store.
+	dir, err := ioutil.TempDir("", "opa_disk_example")
+	check(err)
+
+	// Cleanup temporary directory after finishing.
+	defer os.RemoveAll(dir)
+
+	// Create a new disk-based store.
+	store, err := disk.New(ctx, disk.Options{
+		Dir: dir,
+		Partitions: []storage.Path{
+			storage.MustParsePath("/acmecorp/tenants"),
+		},
+	})
+	check(err)
+
+	// Insert data into the store. The `storage.WriteOne` function automatically
+	// opens a write transaction, applies the operation, and commits the
+	// transaction in one-shot.
+	err = storage.WriteOne(ctx, store, storage.AddOp, storage.MustParsePath("/"), util.MustUnmarshalJSON([]byte(`{
+		"acmecorp": {
+			"tenants": {
+				"initech.io": {
```

Comment:
Both of these seem to exist.... should we pick something more fictional, perhaps?